libssh-rs/src/error.rs

```diff
@@ -53,3 +57,9 @@ impl From<std::ffi::NulError> for Error {
         Error::Fatal(err.to_string())
     }
 }
+
+impl std::fmt::Debug for RequestAuthAgentError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "RequestAuthAgentError({})", self.0)
```

Comment:
I think this will go away as part of my other suggestions, but I wanted to note that this is more correctly written as `f.debug_struct("RequestAuthAgentError").field(&self.0).finish()`

Reply:
Done, thanks!